src/components/Header/HeaderUserMenu/HeaderUserMenu.test.js

```diff
@@ -71,4 +77,26 @@ describe('Header user menu', () => {
     );
     expect(getByText('Upgrade')).toBeInTheDocument();
   });
+
+  it('user have SMS enabled', () => {
+    const { getByText } = render(
+      <IntlProvider>
+        <HeaderUserMenu user={userData} />
+      </IntlProvider>,
+    );
+    expect(getByText('Available for SMS')).toBeInTheDocument();
```

Comment:
How about here validating against the variable? `userData.sms.description`
```suggestion
    expect(getByText(userData.sms.description)).toBeInTheDocument();
```